### PR TITLE
Added Vacuum

### PR DESCRIPTION
**Pull request by nicandris**
hey, notice the sendToDock command. not sure if it's correctly in there or not, but it works 😬

### CODE REVIEW

File: src/http/services/sync.service.ts [hunk 1]
```diff
@@ -109,6 +109,13 @@ export class SyncService {
                         sync.type = DeviceTypes.Lock;
                         sync.traits.push(Traits.LockUnlock);
                         break;
+                    case 'vacuum':
+                        sync.type = DeviceTypes.Vacuum;
+                        sync.traits.push(Traits.StartStop, Traits.Dock);
+                        sync.attributes = {
+                            pausable: true
```
Comment (andrei-tatar) [inline src/http/services/sync.service.ts]: shouldn't this param come from the device property? I noticed there is an optional property called pausable.

File: src/services/devices.repository.ts [hunk 2]
```diff
@@ -105,6 +105,34 @@ export class DevicesRepository {
         DevicesRepository.commands.next({ type: 'activate-scene', uid: this.uid, deviceIds, group, deactivate });
     }
 
+    sendToDock(group: string, deviceIds: string[], { notifyClient = false, requestId }: { notifyClient?: boolean, requestId?: string } = {}) {
```
Comment (andrei-tatar) [inline src/services/devices.repository.ts:108]: why do we need to handle this differently? It seems like a normal state change... isn't it?

Comment (nicandris) [reply src/services/devices.repository.ts:108]: there is no docked state. thats why im handling it like the scene command. we dont send a response back

**Comment by nicandris** [comment]
Screenshot_20200530-150759
Screenshot_20200530-150805
Screenshot_20200530-150811

**Pull request closed without merging**